packages/integration-karma/test/light-dom/synthetic-shadow/x/light/light.html

```diff
@@ -0,0 +1,5 @@
+<template>
+  <h1>I am light DOM</h1>
+  <div lwc:dom="manual"></div>
```

Comment:
Once we implement the light DOM template directive, this template should throw at compile time. The `lwc:dom="manual"` directive has no meaning for light DOM and shouldn't be used.

Reply:
OK, I'll remove this. I may as well also remove the `.appendChild()` test (to ensure that `lwc-dom="manual"` is not actually having any effect.